source/_integrations/conversation.markdown

```diff
@@ -167,7 +167,7 @@ Send a message to a conversation agent for processing.
 | `language`                 | yes      | Language of the text |
 | `agent_id`                 | yes      | ID of conversation agent to use |
 
-This service is able to return a response value. The response is the same response as for the
+This service is able to return [response data](https://www.home-assistant.io/docs/scripts/service-calls/#use-templates-to-handle-response-data). The response is the same response as for the
```

Comment:
This link is not correct, is is a full link and should have been a link relative from the root.

If this was tested... you would have noticed you'd be navigating away from your development environment.

Please create an PR to correct this.

../Frenck

/CC @c0ffeeca7

Reply:
Thank you! Will create a PR to fix it.